.github/workflows/test.yaml

```diff
@@ -43,13 +43,21 @@ jobs:
       - name: Prepare build
         run: yarn e2e:pre-prod
       
-      - name: Run tests
+      - name: Run admin user tests
         run: |
-          yarn e2e:prod
+          GREP_TAGS=@adminUser TEST_USERNAME=admin yarn e2e:prod
           mkdir -p coverage-artifacts/coverage
           cp coverage/e2e/coverage-final.json coverage-artifacts/coverage/coverage-e2e.json
 
+      - name: Run standard user tests
+        if: ${{ success() || failure() }}
+        run: |
+          GREP_TAGS=@standardUser TEST_USERNAME=standard_user yarn e2e:prod && yarn docker:local:stop
+          mkdir -p coverage-artifacts/coverage
+          cp coverage/e2e/coverage-final.json coverage-artifacts/coverage/coverage-e2e.json
+        
       - name: Upload coverage
+        if: ${{ success() || failure() }}
```

Comment:
shouldn't this step only happen on success (which i think is the case when `if` is missing?

Reply:
I thought we wanted it to happen in either case but maybe I'm just misunderstanding the purpose of this step. I'll removed that line of code